scripts/upgrade

```diff
@@ -7,23 +7,22 @@
 source _common.sh
 source /usr/share/yunohost/helpers
 
-ynh_app_setting_set_default --key=php_upload_max_filesize --value=256M
+ynh_app_setting_set_default --key=php_upload_max_filesize --value=2G
 
-ynh_app_setting_set_default --key=php_memory_limit --value=256M
+ynh_app_setting_set_default --key=php_memory_limit --value=2G
 
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
 ynh_script_progression "Upgrading source files..."
 
-ynh_setup_source --dest_dir="$install_dir" --keep=".env"
+ynh_setup_source --dest_dir="$install_dir"
 
-#=================================================
-# UPDATE A CONFIG FILE
-#=================================================
-# ynh_script_progression "Updating configuration..."
-
-# ynh_config_add --template=".env.example" --destination="$install_dir/.env"
+mv $install_dir/index.php $install_dir/www/index.php -f
+mv $install_dir/app/ $install_dir/www/app/ -f
+mv $install_dir/directory-lister.svg $install_dir/www/directory-lister.svg -f
+mv $install_dir/LICENSE $install_dir/www/LICENSE -f
+mv $install_dir/README.md $install_dir/www/README.md -f
```

Comment:
What about using full replace option?
`ynh_setup_source --dest_dir="$install_dir/www" --full_replace`

Reply:
This avoids deleting the uploaded data. But when I see it now, I think `ynh_setup_source --dest_dir="$install_dir/www" --full_replace` will be better since it will not destroy uploaded files